### DISCUSSION TITLE
Add GitHub Gists to mobile natively

### DISCUSSION THREAD

**Discussion by Moosems** [community/community#55524] [Mobile] [answered]
I believe that GitHub Mobile could have another item in the menu bar at the bottom allowing people to view their Gists and comment on them.

**Comment by aconsuegra** [comment, marked as answer]
Hi @Moosems, thanks for your feedback! I'm going to close this discussion as duplicate of https://github.com/orgs/community/discussions/5921. Feel free to upvote that one.
